src/Management/src/Endpoint/Security/EndpointClaim.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Steeltoe.Management.Endpoint.Security;
 
-public class EndpointClaim
+public sealed class EndpointClaim
```

Comment:
This is used in the options type and although it would work, it might mislead users to assume that we support all the other remaining members which we dont

Reply:
Good point, I didn't realize that.